apollo-router/src/plugins/snapshots/apollo_router__plugins__expose_query_plan__tests__it_expose_query_plan-2.snap

```diff
@@ -127,36 +127,36 @@ expression: "serde_json::to_value(response).unwrap()"
                     "@",
                     "reviews",
                     "@",
-                    "product"
+                    "author"
                   ],
                   "node": {
                     "kind": "Fetch",
-                    "serviceName": "products",
+                    "serviceName": "accounts",
                     "requires": [
                       {
                         "kind": "InlineFragment",
-                        "typeCondition": "Product",
+                        "typeCondition": "User",
```

Comment:
This _looks_ like a difference only in ordering in the child nodes of a `Parallel` node, which is we consider not significant. (`both` mode comparison specifically ignores ordering in `vec_matches_result_as_set()`)